app/src/test/java/ca/llamabagel/transpo/ui/trips/GetStopUseCaseTest.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2019 Derek Ellis. Subject to the MIT license.
+ */
+
+package ca.llamabagel.transpo.ui.trips
+
+import ca.llamabagel.transpo.data.Result
+import ca.llamabagel.transpo.data.TestStops
+import ca.llamabagel.transpo.data.provideFakeTripsRepository
+import kotlinx.coroutines.runBlocking
+import org.junit.Assert.*
+import org.junit.Test
+
+class GetStopUseCaseTest {
+
+    private val getStop = GetStopUseCase(provideFakeTripsRepository())
+
+    @Test
+    fun `get stop`() = runBlocking {
+        val stop = getStop(TestStops.mackenzieKing1A.id)
+
+        assertTrue(stop is Result.Success)
+        assertEquals(TestStops.mackenzieKing1A, (stop as Result.Success).data)
```

Comment:
I think this should be two separate tests with more descriptive names. Each test should only have 1 assertion.

Reply:
Both assertions are verifying the same thing. Removing the first assertion wouldn't change the outcome of the test since the second assertion would throw an exception if it weren't a `Result.Success`. It's just explicitly verified there.

There aren't strict rules about only having a single assertion per test. Personally, I think that as long as they're verifying the same thing, then it's fine. It'd be tedious to write two tests just to check `is Result.Success` and then `is correct data`.

Reply:
http://www.owenpellegrin.com/blog/testing/how-do-you-solve-multiple-asserts/
If removing the first assertion wouldn't change anything.. then maybe we should just remove it?